source/drivers/php.txt

```diff
@@ -99,7 +111,11 @@ How to get help
 - Join our `Google Group <https://groups.google.com/forum/#!forum/mongodb-user>`__.
 - Ask on `Stack Overflow <https://stackoverflow.com/questions/tagged/mongodb+php>`__.
 - Visit our `Support Channels <https://docs.mongodb.com/manual/support/>`__.
-- See our `JIRA <https://jira.mongodb.org/projects/PHP/summary>`__ to raise issues or request features.
+- See our JIRA to raise issues or request features.
+
+  - `Extension <https://jira.mongodb.org/projects/PHPC/summary>`__
+
+  - `Library <https://jira.mongodb.org/projects/PHPLIB/summary>`__
```

Comment:
Thanks for linking to these pages! I just realized that the descriptions in JIRA still report these as experimental projects and actually point users to use the unmaintained legacy driver! Will fix that immediately.